### PR TITLE
Feature/endpoints configurability

### PR DESCRIPTION
**Pull request by kikkomep**
This PR allows to configure: 
* the hostname and port of the web server hosting the portal;
* TSI & AAP endpoints.

It also adds a template of `proxy.config.json` file useful to setup a development environment.

### CODE REVIEW

File: src/app/shared/service/wiki/wiki.service.ts [hunk 1]
```diff
@@ -9,19 +9,16 @@ import { AppConfig } from '../../../app.config';
 @Injectable()
 export class WikiService {
   private gitRepoName = 'phenomenal-h2020.wiki';
-  private baseUrl = '';
-  private metadataUrl: string;
+  private baseUrl = window.location.protocol + '//';
+  private metadataUrl = '/php-phenomenal-portal-wiki';
```
Comment (pcm32) [inline src/app/shared/service/wiki/wiki.service.ts]: So these changes won't interfere with the what happens in this line for instance: https://github.com/phnmnl/container-phenomenal-portal/blob/master/setup_backend_host.sh#L8? Just wondering, I'm not an expert in this codebase.

Comment (pcm32) [reply src/app/shared/service/wiki/wiki.service.ts]: ahhh, I see how it works now... nevermind!

**Comment by pcm32** [comment]
Maybe we should start going for squash merges now that they are part of github's offer.

File: src/app/shared/service/wiki/wiki.service.ts (hunk 1 above)
Comment (kikkomep) [reply src/app/shared/service/wiki/wiki.service.ts]: 👍

**Pull request merged**